decision/ledger.go

```diff
@@ -63,10 +65,19 @@ type debtRatio struct {
 	BytesRecv uint64
 }
 
+// Value returns the debt ratio, sent:receive.
 func (dr *debtRatio) Value() float64 {
 	return float64(dr.BytesSent) / float64(dr.BytesRecv+1)
 }
 
+// Score returns the debt _score_ on a 0-1 scale.
+func (dr *debtRatio) Score() float64 {
+	if dr.BytesRecv == 0 {
+		return 0
+	}
+	return float64(dr.BytesRecv) / float64(dr.BytesRecv+dr.BytesSent)
```

Comment:
Is it possible for the denominator to be zero here?

Reply:
No. BytesSent is always >= 0 and we've just checked if BytesRecv is non-zero.